packages/block-editor/src/components/block-breadcrumb/README.md

```diff
@@ -0,0 +1,26 @@
+# Block Breadcrumb
+
+The block breadcrumb trail displays the hierarchy of the current block selection as a breadcrumb. It also allows, using this hierarchy, to navigate to the parent elements of the current block selection. It is located at the very bottom of the editor interface.
+
+![Button block breadcrumb](https://make.wordpress.org/core/files/2020/08/gutenberg-button-block-breadcrumb.png)
+
+![Image in column block breadcrumb](https://make.wordpress.org/core/files/2020/08/gutenberg-image-in-column-block-breadcrumb.png)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+
+
+## Development guidelines
+
+### Usage
+
+Renders a block breadcrumb with default style.
+
+```jsx
+import { BlockBreadcrumb } from '@wordpress/block-editor';
+
+<div className="add-your-class-name">
+    <BlockBreadcrumb />
+</div>
+```
```

Comment:
I think most components in the `block-editor` package including this one need to be embedded in the React tree under `BlockEditorProvider`. I wonder if it's worth mentioning in the READMEs of the different components or not.

Reply:
Yes, I think we should include in each example, because a developer may approach any page without seeing others, the more we can do to make the examples "just work" when copy-and-paste the better.

The example could be just updated to, if you want to include the `<div>` it'd be fine too, but probably not necessary.

```
import { 
  BlockBreadcrumb,
  BlockEditorProvider  } from '@wordpress/block-editor';

<BlockEditorProvider>
    <BlockBreadcrumb/>
</BlockEditorProvider>
```

Reply:
That's not sufficient though because using `BlockEditorProvider` is more complex than that. Maybe we can link to its README?

Reply:
Ok, just updated the example with `BlockEditorProvider`.
Is there anything thing else that should be added?

Reply:
Maybe add something in a "Related components" section?

Reply:
I personally think we should revert to your original code example but add a sentence like that somewhere:

```
Block Editor components are components that can be used to compose the UI of your block editor. Thus, they can only be used under a `BlockEditorProvider` in the components tree. 
```

and link the word `BlockEditorProvider` to its README.

Reply:
✅

Reply:
Works for me. The BlockEditorProvider readme has a complete example 👍